manual/manual/refman/exten.etex

```diff
@@ -2,8 +2,7 @@
 %HEVEA\cutname{extn.html}
 
 This chapter describes language extensions and convenience features
-that are implemented in OCaml, but not described in the
-OCaml reference manual.
+that are implemented in OCaml, but not described in the previous chapter.
```

Comment:
I would have a slight preference for being more explicit, with the title of the chapter and a reference to it, because "the previous chapter" is less clear for people who jump to this part of the manual from an URL.